swagger.yaml

```diff
@@ -169,66 +186,99 @@ paths:
         - application/json
       responses:
         '204':
-          description: user is updated
+          description: User updated
+          schema:
+            $ref: '#/definitions/204Response'
         '404':
           description: 'User not found'
           schema:
             $ref: '#/definitions/404Response'
         '400':
-          description: 'Invalid user details'
+          description: 'Bad Request'
           schema:
             $ref: '#/definitions/400Response'
+        '409':
+          description: 'Conflict'
+          schema:
+            $ref: '#/definitions/409Response'
         '500':
           description: 'Internal server error'
           schema:
             $ref: '#/definitions/500Response'
+
+  /oauth/token:
+    post:
+      summary: Login user
+      tags:
+        - Authentication
+      parameters:
+        - name: 'Login'
+          in: 'body'
+          description: 'Enter email and password for login'
+          required: true
+          schema:
+            $ref: '#/definitions/OauthLogin'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/OauthToken'
+        '401':
+          description: 'User is unauthorized'
+          schema:
+            $ref: '#/definitions/404Response'
+        '500':
+          description: 'Internal Server Error'
+          schema:
+            $ref: '#/definitions/500Response'
       security:
         - JWT: []
 
-  /users/oauth/reset-password:
+  /oauth/reset-password:
```

Comment:
We need 401 response for this API, so we can return Unauthorized when provided invalid expired JWT token.